examples/0.6/BRUX00BEL.xml

```diff
@@ -4,7 +4,7 @@ License: CC By 4.0 (https://creativecommons.org/licenses/by/4.0/)
 Copyright: Royal Observatory of Belgium (please acknowledge us and/or cite doi:10.24414/ROB-GNSS-M3G)
 -->
 <geo:GeodesyML xmlns:geo="urn:xml-gov-au:icsm:egeodesy:0.6" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:gco="http://www.isotc211.org/2005/gco" xmlns:gmd="http://www.isotc211.org/2005/gmd" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" gml:id="BRUX00BEL">
-   <geo:metadata>
+  <geo:metadata>
    <!-- GeodesyML file/station identifier -->
    <gmd:MD_Metadata>
    <gmd:fileIdentifier>
```

Comment:
Nitpick: Indentation is a bit inconsistent in this example file. To be consistent with other files in the repository, we would have all xml files, schemas and example documents, indented with 4 spaces.